pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
     "dask<=2024.2.1",
     "fsspec<=2023.6",
     "geopandas>=0.14",
-    "multiscale_spatial_image==0.11.2",
```

Comment:
I"d add >=

Reply:
yeah it was like that, but was just checking what the issue is with pre CI.